packages/i18n/README.md

```diff
@@ -100,28 +106,51 @@ const Application = (props) => (
       if (isLoading) return null;
 
       return (
-        <ConfigureIntlProvider locale={locale} messages={messages}>
+        <IntlProvider locale={locale} messages={messages}>
           ...
-        </ConfigureIntlProvider>
+        </IntlProvider>
       );
     }}
   </AsyncLocaleData>
 );
 ```
 
-### Generating translation files
+### Using a React hook
+
+Similarly to the `<AsyncLocaleData>`, we also expose a React hook.
+
+```js
+import { IntlProvider } from 'react-intl';
+import { useAsyncLocaleData } from '@commercetools-frontend/i18n';
+
+const Application = (props) => {
+  const { isLoading, locale, messages } = useAsyncLocaleData({
+    locale: props.locale,
+    applicationMessages: loadMessages,
```

Comment:
where is `loadMessages` coming from?

Reply:
It's defined in the example above. Didn't want to copy-paste the function again.